kubernetes/loculus/values.yaml

```diff
@@ -1763,6 +1763,5 @@ defaultResources:
     cpu: "20m"
   limits:
     memory: "1Gi"
-    cpu: "20m"
```

Comment:
huh - this isn't allowed - seems like a bug in the schema - but your original `1000m` seems good for now

Reply:
Schema complains: 
<img width="1026" alt="image" src="https://github.com/user-attachments/assets/7381a9eb-dddc-4b86-8cfc-d31ebdc56b39" />

Is that a correct complaint or a false positive complaint?

Reply:
> seems like a bug in the schema